src/i18n/translation/products/toast/en.ts

```diff
@@ -9,4 +9,5 @@ export const toast = {
     errorMessage: "Failed",
     faild: "Your Delete Failed",
   },
+  wantToDelete: "Are you sure that you want to delete product?",
```

Comment:
Move to delete toast above and change to:
`message: "Are you sure that you want to delete product?"`

Reply:
Fixed